lib/matplotlib/tests/test_colorbar.py

```diff
@@ -242,6 +242,14 @@ def test_contour_colorbar():
     fig.colorbar(CS, orientation='vertical')
 
 
+def test_contour_uniformfield_colorbar():
+    # Smoke test for gh#23817
+    fig, ax = plt.subplots()
+    with pytest.warns(Warning) as record:
+        cs = ax.contour([[1, 1], [1, 1]])
+    assert len(record) == 1
```

Comment:
Sorry, the colorbar should still be there (but can be after the assert).

Reply:
Thanks, I removed that on accident and forgot to add it back before pushing. Fixed now. Sorry about that